prisma/migrations/20240704102905_user/migration.sql

```diff
@@ -0,0 +1,11 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "id" TEXT NOT NULL,
+    "phone" TEXT NOT NULL,
+    "name" TEXT NOT NULL,
```

Comment:
hmm, I don't remember `phone` being in the requirements 🤔 Same goes for `name`. I think you will need `email` and `password` fields.